invenio_records_rest/errors.py

```diff
@@ -29,15 +29,169 @@
 from invenio_rest.errors import RESTException
 
 
+#
+# Search
+#
 class MaxResultWindowRESTError(RESTException):
     """Maximum number of results passed."""
 
     code = 400
-    description = "Maximum number of results have been reached."
+
+    def __init__(self, max):
```

Comment:
IMHO you don't even need to show the max and just keep the original description.